src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala

```diff
@@ -1,19 +1,25 @@
 package org.broadinstitute.dsde.workbench.sam
 package api
 
+import akka.actor.ActorSystem
```

Comment:
I think all changes in these 2 files are due to #221 (comment), which can't be avoided since this PR is to add auth domain check, which means we need to load resource for the policy we're checking, but previously in tests, we didn't care about the resource that the policy belongs to.

I don't think it'll break backward compatibility since in real use case, we definitely have resource pre-created before we have policy.